src/lib/reference.ts

```diff
@@ -26,14 +29,22 @@ export const reference = (
   }
 
   for (const route of metadata.routes ?? []) {
+    const k = `api${route.path}/README.md`
+    files[k] = {
+      contents: Buffer.from('\n'),
+    }
+    const file = files[k] as unknown as File
+    file.layout = 'api-resource.hbs'
```

Comment:
Let's name this `api-route.hbs` now, and use `setApiRouteTemplateContext` etc.
```suggestion
    file.layout = 'api-route.hbs'
```

Reply:
6132664bbb9e1bd61c32e5c7108bd7b931e6ca65